src/components/layout/navigation/Category.tsx

```diff
@@ -0,0 +1,38 @@
+import * as React from "react";
+import {useState} from "react";
+import Tabs from "@mui/material/Tabs";
+import Tab from "@mui/material/Tab";
+import {Paper} from "@mui/material";
+
+export const Category = () => {
+    const [value, setValue] = useState(3)
+    const categories = ['All', 'Category 1', 'Category 2', 'Node.js', 'Category 4']
```

Comment:
all constants and helpers should be initialized outside component. otherwise they will be re-created for every single render